src/gleam/string.gleam

```diff
@@ -73,10 +73,25 @@ if javascript {
 /// ```
 ///
 pub fn reverse(string: String) -> String {
-  string
-  |> string_builder.from_string
-  |> string_builder.reverse
```

Comment:
This method is still buggy for JavaScript right?

Reply:
It works on runtimes that have correct support for graphemes while it fails on those that do not. I think this is acceptable

Reply:
Sorry I was not clear I mean `string_builder.reverse` is still working on UTF-16 chars because the underlying JavaScript methods isn't using graphemes

Reply:
@rockwotj I have voiced the same concern, I will make a separate demo draft/PR with a failing test.

Reply:
Draft/PR with breaking tests: https://github.com/gleam-lang/stdlib/pull/300

Reply:
This looks correct, but if you fix the string builder method then this is fixed without changes here